plugins/flashrom/fu-plugin-flashrom.c

```diff
@@ -51,6 +56,13 @@ fu_plugin_startup (FuPlugin *plugin, GError **error)
 
 	/* we need flashrom from the host system */
 	data->flashrom_fn = fu_common_find_program_in_path ("flashrom", &error_local);
+	if (flashrom_init(1)) {
```

Comment:
What the `1` argument says? Is it needed?

Reply:
maybe I should change to `TRUE` - this argument if is not equal `0`, init performs `selfcheck()`

Reply:
Please do, it looks weird now